components/create_post/create_post.jsx

```diff
@@ -31,6 +31,14 @@ import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx'
 
 const KeyCodes = Constants.KeyCodes;
 
+const isErrorInvalidSlashCommand = (error) => {
+    if (!(error && error.server_error_id)) {
+        return false;
+    }
+
+    return error.server_error_id === 'api.command.execute_command.not_found.app_error';
+};
+
```

Comment:
Please move this function into a separate file, probably to "utils/post_utils.jsx" (or to "components/create_post/create_post_utils.js" if not to be shared) and have it exported.  The idea is to separate business/utility function outside the component and make it easily tested.

Also, can we change the condition like,
```javascript
    if (error && error.server_error_id) {
        return error.server_error_id === 'api.command.execute_command.not_found.app_error';
    }

    return false;
```

Reply:
I agree that this should be in a utility function.

As for the error code, I don't see anywhere in the webapp repo where we are storing these error codes as constants, so I will be keeping the error code string hardcoded.

Reply:
👍, thanks